R CMD CHECK ERROR and NOTE

https://github.com/r-universe/pharmaverse/actions/runs/10861236954

* R-4.5-win
  ```
  * checking DESCRIPTION meta-information ... NOTE
  License stub is invalid DCF.
  ```

* R-4.5-linux
  (note as above)

* R-4.4-win
  (note as above)

* R-4.4-mac
  ```
    ══ Failed tests ════════════════════════════════════════════════════════════════
  ── Failure ('test-h_tidy_pool.R:27:3'): h_tidy_pool is produced correctly ──────
  `result` (`actual`) not identical to `expected` (`expected`).
  
  actual vs expected
                       lower_cl_est
    actual[1, ]   -2.57577141468279
  - actual[2, ]   -2.6453193050416 
  + expected[2, ] -2.64531930504159
  
  `actual$lower_cl_est`:   "-2.57577141468279" "-2.6453193050416" 
  `expected$lower_cl_est`: "-2.57577141468279" "-2.64531930504159"
  ── Failure ('test-tidypool.R:41:3'): tidy.pool is produced correctly ───────────
  `result` (`actual`) not identical to `expected` (`expected`).
  
  actual vs expected
                       lower_cl_est
    actual[1, ]   -2.57577141468279
  - actual[2, ]   -2.6453193050416 
  + expected[2, ] -2.64531930504159
    actual[3, ]   -5.52033399517686
    actual[4, ]   -4.15418494506937
    actual[5, ]   -7.77106696884431
  
      actual$lower_cl_est | expected$lower_cl_est    
  [1] "-2.57577141468279" | "-2.57577141468279"   [1]
  [2] "-2.6453193050416"  - "-2.64531930504159"   [2]
  [3] "-5.52033399517686" | "-5.52033399517686"   [3]
  [4] "-4.15418494506937" | "-4.15418494506937"   [4]
  [5] "-7.77106696884431" | "-7.77106696884431"   [5]
  
  [ FAIL 2 | WARN 0 | SKIP 1 | PASS 4 ]
  ```

* R-4.3-mac
  (test failed as above)

If this is a fixable note - please add this to the r-cmd-check notes blocklist in all of our repos

A new error for `rhub`:
```
── R CMD build ─────────────────────────────────────────────────────────────────
* checking for file ‘.../DESCRIPTION’ ... OK
* preparing ‘tern.rbmi’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
--- re-building ‘tern-rbmi.Rmd’ using rmarkdown
Quitting from lines 156-164 [unnamed-chunk-7] (tern-rbmi.Rmd)
Error: processing vignette 'tern-rbmi.Rmd' failed with diagnostics:
Boost not found; call install.packages('BH')
--- failed re-building ‘tern-rbmi.Rmd’
SUMMARY: processing the following file failed:
  ‘tern-rbmi.Rmd’
Error: Vignette re-building failed.
Execution halted
Error in proc$get_built_file() : Build process failed
Calls: <Anonymous> ... build_package -> with_envvar -> force -> <Anonymous>
Execution halted
Error: Process completed with exit code 1.
```

--- a/vignettes/tern-rbmi.Rmd
+++ b/vignettes/tern-rbmi.Rmd
@@ -20,7 +20,7 @@ editor_options:
 suggested_dependent_pkgs <- c("rstan")
 knitr::opts_chunk$set(
   collapse = TRUE,
-  comment = "#>"
+  comment = "#>",
   eval = all(vapply(
     suggested_dependent_pkgs,
     requireNamespace,

--- a/vignettes/tern-rbmi.Rmd
+++ b/vignettes/tern-rbmi.Rmd
@@ -17,11 +17,19 @@ editor_options:
 ---
 
 ```{r setup, include = FALSE}
+suggested_dependent_pkgs <- c("rstan")
 knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>"
+  eval = all(vapply(
+    suggested_dependent_pkgs,
+    requireNamespace,
+    logical(1),
+    quietly = TRUE
+  ))
 )
 ```
+
 
 ## Introduction to tern.rbmi
 
@@ -133,7 +141,7 @@ The `rbmi::draws()` function fits the imputation models and stores the correspon
 
 #### Define key variables
 
-Define the names of key variables in our dataset and the covariates included in 
+Define the names of key variables in our dataset and the covariates included in
 the imputation model using `rbmi::set_vars()`. Note that the covariates argument
 can also include interaction terms.
 
@@ -149,7 +157,7 @@ draws_vars <- rbmi::set_vars(
 draws_vars$subjid <- "TMP_ID"
 ```
 
-Define which imputation method to use, then create samples for the imputation 
+Define which imputation method to use, then create samples for the imputation
 parameters by running the `draws()` function.
 
 ```{r}

